ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -574,19 +594,21 @@
               en: 'Knockback Cleave; Face Outside',
               de: 'Rückstoß Cleave; nach Außen schauen',
               ja: 'ノックバック ソード; 外向く',
+              fr: 'Poussée Cleave; Regarde à l\'extérieur',
             };
           }
           return {
             en: 'Knockback Charge; Face Middle',
             de: 'Rückstoß Charge; zur Mitte schauen',
             ja: 'ノックバック チャージ; 中央向く',
+            fr: 'Poussée Charge; Regarde à l\'intérieur',
           };
         }
         if (isOddNumber) {
           return {
             en: 'Knockback Cleave on YOU',
             de: 'Rückstoß Cleave auf DIR',
-            fr: 'Cleave sur vous',
+            fr: 'Cleave sur TOI',
```

Comment:
Can you explain this change to me? Mostly because other folks doing french translations have been using vous instead of toi everywhere, and so it'd be nice to be consistent.  Maybe worth leaving as vous for now here (and elsewhere) and then chatting with @mooondark at least about what's best to follow up in the future?

Reply:
This is hard to explain. In french 'you' is splitted in two parts: 'tu' when used for a single person or familiar form and 'vous' for two (or more) persons or as the polite form.

I think 'vous' is far better for cactbot. So in order to be consistant, we should keep 'vous' when translating 'you'.

Reply:
@quisquous @mooondark Both are correct, and using vous for consistency is absolutely valid. I actually prefer “vous” myself as well but could have sworn I saw “TOI” in other places and wanted to keep it consistent. Vous is better though so please overrule that one.

Reply:
I’ll keep in mind to use vous for other translations!